benchmarks/benchmark_functions/pl-braket-pipeline.py

```diff
@@ -144,3 +145,63 @@ def circuit(params):
         return [qml.sample(qml.PauliZ(i)) for i in range(n_wires)]
 
     circuit(params)
+
+
+def benchmark_qchem(dev_name, s3=None):
+    """A basic qchem workflow
+
+    Args:
+        dev_name (str): Either "local", "sv1", "tn1", or "ionq"
+        s3 (tuple):  A tuple of (bucket, prefix) to specify the s3 storage location
+    """
+
+    if dev_name == "local":
+        n_wires = 4
+        device = qml.device("braket.local.qubit", wires=n_wires, shots=None)
+    elif dev_name == "sv1":
+        n_wires = 4
+        device = qml.device(
+            "braket.aws.qubit",
+            device_arn="arn:aws:braket:::device/quantum-simulator/amazon/sv1",
+            s3_destination_folder=s3,
+            wires=n_wires,
+            shots=None,
+        )
+    elif dev_name == "tn1":
+        shots = 1000
+        n_wires = 4
+        device = qml.device(
+            "braket.aws.qubit",
+            device_arn="arn:aws:braket:::device/quantum-simulator/amazon/tn1",
+            s3_destination_folder=s3,
+            wires=n_wires,
+            shots=shots,
+        )
+    elif dev_name == "ionq":
+        shots = 100
+        n_wires = 4
+        device = qml.device(
+            "braket.aws.qubit",
+            device_arn="arn:aws:braket:::device/qpu/ionq/ionQdevice",
+            s3_destination_folder=s3,
+            wires=n_wires,
+            shots=shots,
+        )
+    else:
+        raise ValueError("dev_name not 'local', 'sv1','tn1', or 'ionq'")
+
+    def circuit(params, wires):
+        qml.BasisState(np.array([1, 1, 0, 0], requires_grad=False), wires=wires)
+        for i in wires:
+            qml.Rot(*params[i], wires=i)
+        qml.CNOT(wires=[2, 3])
+        qml.CNOT(wires=[2, 0])
+        qml.CNOT(wires=[3, 1])
+
+    np.random.seed(0)
```

Comment:
Looking above, it looks like PennyLane Numpy is defined as `pnp` in this file, not `np`.  Maybe we should change that.

Also, if we will be using master PL instead of the latest release, we should change to using the updated random number generation method, see Line 72/73 of this file.